### DISCUSSION TITLE
Markup in "My Times" description

### DISCUSSION THREAD

**Discussion by snapet** [kimai/kimai#4879] [Questions & Answers]
Hi community!

I often use Markup in the time sheet entries but I (clearly) don't understand it very well.
I use bullets that are indented but I only manage to have two levels. Is that a standard Markup feature? Or specific to Kimai?

This, two levels description works:
```
* Did some things
  * Did first thing
  * Did second thing
```
And renders as:
* Did some things
  * Did first thing
  * Did second thing

Below gives me three levels here on Github:
```
* Did some things
  * Did first thing
    * But also this little thing
    * And that one
  * Did second thing
```

On Github:
* Did some things
  * Did first thing
    * But also this little thing
    * And that one
  * Did second thing

But in the description in Kimai it renders as:
* Did some things
  * Did first thing
  * But also this little thing
  * And that one
  * Did second thing

If anyone can point me in the right direction or just bluntly tell me that it just doesn't work I would be very grateful.

**Comment by kevinpapst** [comment]
There is likely no CSS for deeper levels. I assume the HTML is correct, but the styling doesn't indent deeper levels.

**Reply by snapet** [reply]
Thanks for your quick reply Kevin!
Is that something I can change and if so, where would that be?
